backends/tfhe-cuda-backend/cuda/include/programmable_bootstrap.h

```diff
@@ -41,17 +41,15 @@ void cuda_programmable_bootstrap_amortized_lwe_ciphertext_vector_32(
     void *lwe_array_in, void *lwe_input_indexes, void *bootstrapping_key,
     int8_t *pbs_buffer, uint32_t lwe_dimension, uint32_t glwe_dimension,
     uint32_t polynomial_size, uint32_t base_log, uint32_t level_count,
-    uint32_t num_samples, uint32_t num_luts, uint32_t lwe_idx,
-    uint32_t max_shared_memory);
+    uint32_t num_samples, uint32_t max_shared_memory);
```

Comment:
Since we are cleaning, can we also remove `max_shared_memory`?

Reply:
I'll do it in another PR tomrrow :slightly_smiling_face: